platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -978,6 +973,12 @@ public boolean isPermittedToViewFeature(String featureName) {
     }
   }
 
+  private static class FeatureServiceException extends RuntimeException {
```

Comment:
❓ Do we really need a new exception class for this?

Reply:
The alternative is to suppress the finding associated with throwing `RuntimeException`.